internal/configs/parser_test.go

```diff
@@ -45,6 +45,7 @@ func testParser(files map[string]string) *Parser {
 // module and returns its configuration. This is a helper for use in unit tests.
 func testModuleConfigFromFile(filename string) (*Config, hcl.Diagnostics) {
 	parser := NewParser(nil)
+	parser.allowExperiments = true // Allow language experiments in tests so we can verify actions.
```

Comment:
testModuleFromDirWithExperiments exists so we don't have to enable this across all tests; it should not be the default behavior